packages/@lightningjs/ui-components/src/components/Tile/Tile.d.ts

```diff
@@ -24,6 +24,7 @@ import Surface, { SurfaceStyle } from '../Surface';
 export type TileStyle = SurfaceStyle & {
   animationEntrance: Record<string, unknown>;
   animationExit: Record<string, unknown>;
+  metadataLocation: 'standard' | 'inset';
   paddingX: number | string;
   paddingY: number | string;
   paddingYProgress: number | string;
```

Comment:
I just realized I never added that new paddingYBetweenContent prop here 🤦‍♀️ would you mind adding it while you're in here please?